src/Symfony/Component/HttpKernel/EventListener/DumpListener.php

```diff
@@ -37,8 +39,16 @@ public function __construct(ClonerInterface $cloner, DataDumperInterface $dumper
         $this->dumper = $dumper;
     }
 
-    public function configure()
+    public function configure(GetResponseEvent $event)
     {
+        // if the request is issued by curl, we do not enable HTML dumper;
+        if (false !== strpos($event->getRequest()->headers->get('user-agent'), 'curl/')) {
```

Comment:
You are right! Forget my comment. I just wondering if there is a way to detect requests made via the CLI, not via a particular user agent.